library/include/hipblas.h

```diff
@@ -17765,6 +17826,48 @@ HIPBLAS_EXPORT hipblasStatus_t hipblasGemmEx_v2(hipblasHandle_t      handle,
                                                 hipblasComputeType_t computeType,
                                                 hipblasGemmAlgo_t    algo);
 
+HIPBLAS_EXPORT hipblasStatus_t hipblasGemmExWithFlags(hipblasHandle_t    handle,
```

Comment:
I only added HIPBLAS_DEPRECATED_MSG for the top-level hipblasDatatype_t, instead of adding it for each function which uses hipblasDatatype_t to avoid excessive warnings. We can discuss if there's a reason to add it for each function as well.

Reply:
OK, deprecation message for top-level hipblasDatatype_t is good.